src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
 
 const ChatEntry = (props) => {
+  const sender = props.sender;
+  const time = props.timeStamp;
+  const body = props.body;
+  const [heart, changeHeart] = useState('🤍');
+  const changeLikeButton = () => {
+    if (heart === '🤍') {
+      changeHeart('❤️');
+    } else {
+      changeHeart('🤍');
+    }
+    props.changeLikeButton(props.id);
+  };
+
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">
+          <TimeStamp time={time} /> years ago
+        </p>
+        <button className="like" onClick={changeLikeButton}>
+          {heart}
```

Comment:
If you remove the heart state variable, you have a few choices here.  One option is to use a ternary operator: `props.liked ? '❤️' : '🤍'`.  Another option is to write a function that returns the correct heart based on the value of props.liked and call the function here.